src/main/scala/tech/cryptonomic/conseil/tezos/TezosNodeOperator.scala

```diff
@@ -266,6 +299,11 @@ class TezosNodeOperator(val node: TezosRPCInterface, val network: String, batchC
   def getDelegatesForBlocks(keysBlocksIndex: Map[PublicKeyHash, BlockReference]): PaginatedDelegateResults = {
     import TezosTypes.Syntax._
 
+    val reverseIndex =
+      keysBlocksIndex.groupBy { case (pkh, blockRef) => blockRef._1 }
+       .mapValues(_.keySet)
```

Comment:
This method is deprecated in 2.13. We should avoid it.

Reply:
There are 7 places where this is used in the codebase.
Should we do it right now, or maybe define a custom issue with his own very simple PR?
Should we wait for 2.13 to become relevant to Conseil?